pom.xml

```diff
@@ -273,7 +273,7 @@
         <version>1.0.1-SNAPSHOT</version>
         <configuration>
           <cdapArtifacts>
-            <parent>system:cdap-data-pipeline[4.3.0-SNAPSHOT,9.0.0-SNAPSHOT)</parent>
+            <parent>${app.parents}</parent>
```

Comment:
does this produce a config with two element in the parents json array? I thought multiple parents are supposed to be specified as multiple 'parent' elements in this section.

Reply:
oh yeah.. so need to have two `<parent>`?

Reply:
hum.. meaning I can't easily define it in the property.